nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/scheduler/SchedulerServiceImpl.java

```diff
@@ -108,9 +109,13 @@ protected void setupScheduler() throws IOException, SchedulerException {
         // https://jira.nuxeo.com/browse/NXP-7303
         GroupMatcher<JobKey> matcher = GroupMatcher.jobGroupEquals("nuxeo");
         Set<JobKey> jobs = scheduler.getJobKeys(matcher);
-        scheduler.deleteJobs(new ArrayList<JobKey>(jobs));
-        for (Schedule each : registry.getSchedules()) {
-            registerSchedule(each);
+        try {
+            scheduler.deleteJobs(new ArrayList<JobKey>(jobs));
+            for (Schedule each : registry.getSchedules()) {
+                registerSchedule(each);
+            }
+        } catch (LockException cause) {
+            log.warn("scheduler already re-initializing, another cluster node concurrent startup ?", cause);
         }
```

Comment:
I'd say no, but this is worth a comment explaining at which point (delete ? first schedule ?) the LockException is being thrown.